Add Spanish translation of README.md (#6)
santisoler/cc-licenses: es/README.md

# Licencias Creative Commons para proyectos de Github

[![Main](https://img.shields.io/badge/main%20language-EN-blue)](/../../)

En este repositorio podrás encontrar formas sencillas de aplicar Licencias Creative Commons
en repositorios de Github a través del lenguaje Markdown.

> **ADVERTENCIA:**
> No debes utilizar licencias Creative Commons para software.
> Usa licencias de software libre para ese tipo de repositorios, como GPL, BSD, etc.
> Las licencias de Creative Commons son solo para trabajos intelectuales.

> La manera más facil de encontrar la licencia CC adecuada para tu proyecto es en el

Reviewer: ```suggestion
La manera más fácil de encontrar la licencia CC adecuada para tu proyecto es en el
```

[sitio web Creative Commons](https://creativecommons.org/choose/).

> Aqui te presentamos dos tipos de licencias CC:

Reviewer: ```suggestion
Aquí te presentamos dos tipos de licencias CC:
```

* [Creative Commons Atribución 4.0 Internacional](#cc-atribución-40-internacional)

* [Creative Commons Atribución-CompartirIgual 4.0 Internacional](#cc-atribución-compartirigual-40-internacional)

> Puedes obtener más información sobre licencias y textos de licencias con formato de texto plano

Reviewer: ```suggestion
Puedes obtener más información sobre licencias y sus textos completos en texto plano
```

en https://choosealicense.com/.

Si quieres descargar la versión vectorizada de las imagenes de Creative Commons
y otras, por favor visita https://creativecommons.org/about/downloads/.

## CC Atribución 4.0 Internacional

Para agregar una licencia CC BY a tu proyecto, simplemente agrega lo siguiente a tu
`README.md`. También debes copiar el [archivo de texto](LICENSE-CC-BY) de la licencia correspondiente y
renombrarlo a `LICENSE`.

```markdown
Shield: [![CC BY 4.0][cc-by-shield]][cc-by]

> Este trabajo está licenciado bajo [Licencia Creative Commons Atribución 4.0 Internacional][cc-by].

Reviewer: ```suggestion
Esta obra está bajo una [Licencia Creative Commons Atribución 4.0 Internacional][cc-by].
```

[![CC BY 4.0][cc-by-image]][cc-by]

[cc-by]: https://creativecommons.org/licenses/by/4.0/deed.es
[cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
[cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg
```

Shield: [![CC BY 4.0][cc-by-shield]][cc-by]

> Este trabajo está licenciado bajo [Licencia Creative Commons Atribución 4.0 Internacional][cc-by].

Reviewer: ```suggestion
Esta obra está bajo una [Licencia Creative Commons Atribución 4.0 Internacional][cc-by].
```

[![CC BY 4.0][cc-by-image]][cc-by]

[cc-by]: https://creativecommons.org/licenses/by/4.0/deed.es
[cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
[cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg

## CC Atribución-CompartirIgual 4.0 Internacional
Para agregar una licencia CC BY-SA a tu proyecto, simplemente agrega lo siguiente a tu
`README.md`. También debes copiar el [archivo de texto](LICENSE-CC-BY-SA) de la licencia correspondiente y
renombrarlo a `LICENSE`.

```markdown
Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]

> Este trabajo está licenciado bajo

Reviewer: ```suggestion
Esta obra está bajo una
```

[Licencia Creative Commons Atribución-CompartirIgual 4.0 Internacional][cc-by-sa].

[![CC BY-SA 4.0][cc-by-sa-image]][cc-by-sa]

[cc-by-sa]: https://creativecommons.org/licenses/by-sa/4.0/deed.es
[cc-by-sa-image]: https://licensebuttons.net/l/by-sa/4.0/88x31.png
[cc-by-sa-shield]: https://img.shields.io/badge/License-CC%20BY--SA%204.0-lightgrey.svg
```

Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]

> Este trabajo está licenciado bajo [ Licencia Creative Commons Atribución-CompartirIgual 4.0 Internacional][cc-by-sa].

Reviewer: ```suggestion
Esta obra está bajo una
[Licencia Creative Commons Atribución-CompartirIgual 4.0 Internacional][cc-by-sa].
```

# Licencias Creative Commons para proyectos de Github

[![Main](https://img.shields.io/badge/main%20language-EN-blue)](/../../)
[![Deutsch](https://img.shields.io/badge/translation-DE-red)](de/)

En este repositorio podrá encontrar formas sencillas de aplicar Licencias Creative Commons

> en repositorios Github a través del lenguaje Markdown.

Reviewer: ```suggestion
en repositorios de Github a través del lenguaje Markdown.
```

> **ADVERTENCIA:**

> > No debería usar licencias Creative Commons para software.

Reviewer: ```suggestion
> No debes utilizar licencias Creative Commons para software.
```

> Usa licencias de software libre para ese tipo de repositorios, como GPL, BSD, etc.
> Las licencias de Creative Commons son solo para trabajos intelectuales.

> La manera más facil de encontrar la licencia CC adecuada para su proyecto es en el

Reviewer: ```suggestion
La manera más facil de encontrar la licencia CC adecuada para tu proyecto es en el
```

[sitio web Creative Commons](https://creativecommons.org/choose/).

> Aqui le presentamos dos tipos de licencias CC:

Reviewer: ```suggestion
Aquí te presentamos dos tipos de licencias CC:
```

> * [Creative Commons Atribución/Reconocimiento 4.0 Internacional](#cc-attribution-40-international)

Reviewer: Creative Commons use a shorter version of the license name on [their instruction deeds](https://creativecommons.org/licenses/by/4.0/deed.es): _Creative Commons Atribución 4.0 Internacional_. So, we might want to do the same in order to keep things simpler.
Besides, the link won't work as it is, so I changed that too.
```suggestion
* [Creative Commons Atribución 4.0 Internacional](#cc-atribución-40-internacional)
```

> * [Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0](#cc-attribution-sharealike-40-international)

Reviewer: ```suggestion
* [Creative Commons Atribución-CompartirIgual 4.0](#cc-atribución-compartirigual-40-internacional)
```

> Se puede obtener más información sobre licencias y textos de licencias con formato de texto sin formato

Reviewer: ```suggestion
Puedes obtener más información sobre licencias y textos de licencias con formato de texto plano
```

en https://choosealicense.com/.

> Si quiere descargar la versión vectorizada de las imagenes de Creative Commons
> y otras por favor visite https://creativecommons.org/about/downloads/.

Reviewer: ```suggestion
Si quieres descargar la versión vectorizada de las imagenes de Creative Commons
y otras, por favor visita https://creativecommons.org/about/downloads/.
```

> ## Creative Commons Atribución/Reconocimiento 4.0 Internacional

Reviewer: ```suggestion
## CC Atribución 4.0 Internacional
```

> Para agregar una licencia CC BY a su proyecto, simplemente agregue lo siguiente a su

Reviewer: ```suggestion
Para agregar una licencia CC BY a tu proyecto, simplemente agrega lo siguiente a tu
```

> `README.md`. También debe copiar el [archivo de texto](LICENSE-CC-BY) de la licencia correspondiente y 

Reviewer: ```suggestion
`README.md`. También debes copiar el [archivo de texto](LICENSE-CC-BY) de la licencia correspondiente y 
```

renombrarlo a `LICENSE`.

```markdown
Shield: [![CC BY 4.0][cc-by-shield]][cc-by]

> Este trabajo está licenciado bajo [Creative Commons Atribución/Reconocimiento 4.0 Internacional][cc-by].

Reviewer: El [deed de la licencia](https://creativecommons.org/choose/?lang=es_AR) recomienda utilizar este texto en español:

```suggestion
Esta obra está bajo una [Licencia Creative Commons Atribución 4.0 Internacional][cc-by].
```

[![CC BY 4.0][cc-by-image]][cc-by]

[cc-by]: https://creativecommons.org/licenses/by/4.0/deed.es
[cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
[cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg
```

Shield: [![CC BY 4.0][cc-by-shield]][cc-by]

Este trabajo está licenciado bajo [Creative Commons Atribución/Reconocimiento 4.0 Internacional][cc-by].

[![CC BY 4.0][cc-by-image]][cc-by]

[cc-by]: https://creativecommons.org/licenses/by/4.0/deed.es
[cc-by-image]: https://i.creativecommons.org/l/by/4.0/88x31.png
[cc-by-shield]: https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg

> ## Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0

Reviewer: ```suggestion
## CC Atribución-CompartirIgual 4.0

```

> Para agregar una licencia CC BY-SA a su proyecto, simplemente agregue lo siguiente a su

Reviewer: ```suggestion
Para agregar una licencia CC BY-SA a tu proyecto, simplemente agrega lo siguiente a tu
```

> `README.md`. También debe copiar el [archivo de texto](LICENSE-CC-BY-SA) de la licencia correspondiente y 

Reviewer: ```suggestion
`README.md`. También debes copiar el [archivo de texto](LICENSE-CC-BY-SA) de la licencia correspondiente y 
```

renombrarlo a `LICENSE`.

```markdown
Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]

Este trabajo está licenciado bajo

> [Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0][cc-by-sa].

Reviewer: ```suggestion
Esta obra está bajo una [Licencia Creative Commons Atribución-CompartirIgual 4.0 Internacional][cc-by-sa].
```

[![CC BY-SA 4.0][cc-by-sa-image]][cc-by-sa]

[cc-by-sa]: https://creativecommons.org/licenses/by-sa/4.0/deed.es
[cc-by-sa-image]: https://licensebuttons.net/l/by-sa/4.0/88x31.png
[cc-by-sa-shield]: https://img.shields.io/badge/License-CC%20BY--SA%204.0-lightgrey.svg
```

Shield: [![CC BY-SA 4.0][cc-by-sa-shield]][cc-by-sa]

> Este trabajo está licenciado bajo [Creative Commons Atribución/Reconocimiento-CompartirIgual 4.0][cc-by-sa].

Reviewer: ```suggestion
Esta obra está bajo una [Licencia Creative Commons Atribución-CompartirIgual 4.0 Internacional][cc-by-sa].
```
